tests/unit/EnvironmentContextTest.tsx

```diff
@@ -0,0 +1,116 @@
+import {render, waitFor} from '@testing-library/react-native';
+import React from 'react';
+import {EnvironmentContext, EnvironmentProvider} from '@components/EnvironmentContext';
+import CONST from '@src/CONST';
+
+// Mock getEnvironment and getEnvironmentURL
+const mockGetEnvironment: jest.MockedFunction<() => Promise<string>> = jest.fn();
+const mockGetEnvironmentURL: jest.MockedFunction<() => Promise<string>> = jest.fn();
+
+jest.mock('@libs/Environment/Environment', () => ({
+    getEnvironment: jest.fn().mockImplementation(() => mockGetEnvironment()),
+    getEnvironmentURL: jest.fn().mockImplementation(() => mockGetEnvironmentURL()),
+}));
+
+describe('EnvironmentProvider', () => {
```

Comment:
Can we add some test cases for testing this without wrapping the whole string with HTML tags?

For example, something like this:

```
Read up on <a href="https://help.expensify.com">Expensify Help</a> to find out more.
```

want to make sure `parseDocument(html)` works with only partial HTML